client/src/sids_relay.py

```diff
@@ -44,7 +54,7 @@ def relay(self, frame: AXFrame):
         params = {
             "noradID": self.config.get_conf("Mission Control", "norad-id"),
             "source": self.config.get_conf("Mission Control", "receiver-callsign"),
-            "timestamp": frame.recv_time.isoformat(),
+            "recvTime": frame.recv_time.isoformat(),
```

Comment:
What's the reasoning here for changing this variable? The actual spec sheet calls it "timestamp" too. https://picsat.obspm.fr/sids/doc?locale=en

Secondly, the variable name isn't in snake_case!

Reply:
The endpoint excepted a value called recvTime so I changed it. I gather that we should change the endpoint instead.

Reply:
Yes.